plugins/src/webrtcsink/imp.rs

```diff
@@ -85,6 +85,8 @@ struct WebRTCPad {
     /// The (fixed) caps of the corresponding input stream
     in_caps: gst::Caps,
     /// Our offer caps
+    // FIXME: Use this at some point
+    #[allow(dead_code)]
     caps: gst::Caps,
```

Comment:
Is this intentionally unused currently?

Reply:
No, I think this was unused since the initial commit, it can be removed altogether

Reply:
Ok, removed that one